tool/CMakeLists.txt

```diff
@@ -1,4 +1,12 @@
+# Note: The library can be static (.a) or shared (.so)
+if(NOT DEFINED PROTOBUF_LIBRARIES)
+    message(FATAL_ERROR "PROTOBUF_LIBRARIES is not found")
+endif()
+
 add_executable(onnx_viewer onnx_viewer.cpp)
-target_link_libraries(onnx_viewer menoh)
+
+# ONNX requires protobuf regardless of whether it is static and shared
+target_link_libraries(onnx_viewer ${PROTOBUF_LIBRARIES} menoh)
```

Comment:
Is it safe to link libprotobuf here when `LINK_STATIC_LIBPROTOBUF` is ON?
Doen't this lead to mixing multiple copy of libprotobuf (one inside libmenoh and one in `${PROTOBUF_LIBRARIES}`) ?

Reply:
Hmm, it works in my environment because, maybe, `libprotobuf` embedded in `libmenoh` is hidden and separated from `${PROTOBUF_LIBRARIES}` by using `--version-script`. But we'll remove this file from the repository as mentioned in https://github.com/pfnet-research/menoh/pull/70#discussion_r213880805 anyway.

Reply:
Closing the conversation assuming #77 will be merged.